streams/src/main/java/org/apache/kafka/streams/processor/internals/InternalTopologyBuilder.java

```diff
@@ -121,6 +121,9 @@
 
     private Map<Integer, Set<String>> nodeGroups = null;
 
+    // this is only temporary for 2.0 and should be removed
```

Comment:
Maybe mark it TODO so we can search for it later.

Reply:
acj,